lib/Elastica/Query/MatchQuery.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Elastica\Query;
+
+/**
+ * Match query.
+ *
+ * This class is for forward compatibility reasons. For PHP 8 and above use MatchQuery as Match is reserved.
+ */
+class MatchQuery extends Match
```

Comment:
I think that won't work as `Match` is reserved (even when using extend).

Reply:
It doesn't need to.

6.x only supports PHP ^7.0, so unless you force it, it won't run on PHP 8.0 - https://github.com/ruflin/Elastica/blob/6.x/composer.json#L15

I'm not adding it to add PHP 8.0 support, I'm doing it so people can migrate the code and be able to use newer 6.x versions or 7.x with the MatchQuery class, and not need to have conditional support for the class existing (which can be a pain when you're extending this class, for example)

Reply:
See comments on https://github.com/ruflin/Elastica/issues/1879

It was suggested to avoid backporting all of https://github.com/ruflin/Elastica/pull/1799

Reply:
Thanks, sorry I didn't understand at first glance ;)